distributed/comm/tests/test_ucx.py

```diff
@@ -169,7 +168,12 @@ async def test_ucx_deserialize():
         lambda cudf: cudf.DataFrame([1]).head(0),
         lambda cudf: cudf.DataFrame([1.0]).head(0),
         lambda cudf: cudf.DataFrame({"a": []}),
-        lambda cudf: cudf.DataFrame({"a": ["a"]}).head(0),
+        pytest.param(
+            lambda cudf: cudf.DataFrame({"a": ["a"]}).head(0),
+            marks=pytest.mark.skip(
+                reason="This test segfaults for some reason. So skip running it entirely."
+            ),
+        ),
```

Comment:
This started segfaulting for me. Not sure why. I don't think it is related to Distributed (maybe a cuDF issue?). For now am just disabling this test until we can figure out why/fix it.

cc @kkraus14 @shwina (for vis)

Reply:
Hmm this is news to me but is likely problematic. We'll have to poke around a little bit.

Reply:
There is also another case in this test (shown below) that has been marked as failing for a while as well. Not sure if they are related.

https://github.com/dask/distributed/blob/1747ae1daff5357701aa96ce7d94b30a5596e3c6/distributed/comm/tests/test_ucx.py#L180-L185

Reply:
I'm curious, did this fail before the recent handshake PR?

Reply:
I haven't seen it in UCX-Py's CI, but did run into it locally. Maybe there's something more subtle at play like environment differences. In any event, I don't think it is handshake related.